### PR TITLE
Fix TopDownBottomUpOfLoadedCapture E2E test

### PR DESCRIPTION
**Pull request by florian-kuebler**
In #3436 we adjusted the call tree views, but did not touched the E2E
tests. Now this change fixes the tests and added a testcase for
the unwinding error type to the top-down view.

Bug: http://b/225793930
Test: Run E2E test locally

### CODE REVIEW

File: contrib/automation_tests/test_cases/top_down_tab.py [hunk 1]
```diff
@@ -119,7 +119,40 @@ def _verify_rows_when_thread_expanded(self, tree_view_table):
             ["OrbitTest (all threads)", "100.00% (1583)", "", "", "", ""],
             ["OrbitThread_203 [20301]", "79.34% (1256)", "", "", "", ""],
             ["clone", "79.22% (1254)", "0.00% (0)", "99.84%", "libc-2.24.so", "0x7fe86eff5900"],
-            ["[unwind errors]", "0.13% (2)", "", "0.16%", "", ""],
+            ["[Unwind errors]", "0.13% (2)", "", "0.16%", "", ""],
+        ]
+        self._verify_first_rows(tree_view_table, expectations)
+        logging.info("Verified content of top-down view when one thread is expanded")
+
+    def _verify_unwinding_errors_when_thread_expanded(self, tree_view_table):
```
Comment (dpallotti) [inline contrib/automation_tests/test_cases/top_down_tab.py:127]: Isn't part of this method a duplication of `_verify_rows_when_thread_expanded`? Can this method just be merged into `_verify_rows_when_thread_expanded` or this there a reason to have two?

Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/top_down_tab.py:127]: My intention was to have the pieces separated, but I can merge them if you wish.

Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/top_down_tab.py]: Merged the two methods.

File: contrib/automation_tests/test_cases/top_down_tab.py [hunk 2]
```diff
@@ -119,7 +119,40 @@ def _verify_rows_when_thread_expanded(self, tree_view_table):
...
+        logging.info("Expanding a thread of the top-down view")
```
Comment (dpallotti) [inline contrib/automation_tests/test_cases/top_down_tab.py:118]: This is wrong, you are expanding the "[Unwind errors]" node here.

File: contrib/automation_tests/test_cases/top_down_tab.py [hunk 3]
```diff
@@ -119,7 +119,40 @@ def _verify_rows_when_thread_expanded(self, tree_view_table):
...
+        tree_view_table.get_item_at(3, 0).double_click_input()
+        expected_first_thread_child_count = 2
+        expected_second_thread_child_count = 1
+        expected_total_child_count = \
+            self.EXPECTED_THREAD_COUNT + expected_first_thread_child_count + expected_second_thread_child_count
+        self._verify_row_count(tree_view_table, expected_total_child_count)
+
+        tree_view_table.get_item_at(4, 0).double_click_input()
```
Comment (dpallotti) [inline contrib/automation_tests/test_cases/top_down_tab.py:124]: Can you add another "logging.info" to this for consistency?

File: contrib/automation_tests/test_cases/top_down_tab.py [hunk 4]
```diff
@@ -108,18 +108,39 @@ def _verify_rows_when_tree_collapsed(self, tree_view_table):
...
         tree_view_table.get_item_at(1, 0).double_click_input()
         expected_first_thread_child_count = 2
-        self._verify_row_count(tree_view_table,
-                               self.EXPECTED_THREAD_COUNT + expected_first_thread_child_count)
+        expected_total_child_count = self.EXPECTED_THREAD_COUNT + expected_first_thread_child_count
+        self._verify_row_count(tree_view_table, expected_total_child_count)
+
+        logging.info("Expanding a thread of the top-down view")
+        tree_view_table.get_item_at(3, 0).double_click_input()
+        expected_second_thread_child_count = 1
```
Comment (dpallotti) [inline contrib/automation_tests/test_cases/top_down_tab.py:120]: You did not expand a thread here, I think this variable should be renamed.

File: contrib/automation_tests/test_cases/top_down_tab.py [hunk 5]
```diff
@@ -108,18 +108,39 @@ def _verify_rows_when_tree_collapsed(self, tree_view_table):
...
+        expected_total_child_count += expected_second_thread_child_count
+        self._verify_row_count(tree_view_table, expected_total_child_count)
+
+        tree_view_table.get_item_at(4, 0).double_click_input()
+        expected_third_thread_child_count = 1
```
Comment (dpallotti) [inline contrib/automation_tests/test_cases/top_down_tab.py:125]: Same here, you did not expand a thread, but the error node.

File: contrib/automation_tests/test_cases/top_down_tab.py (hunk 2 above)
Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/top_down_tab.py]: Arg. Thanks!

File: contrib/automation_tests/test_cases/top_down_tab.py (hunk 4 above)
Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/top_down_tab.py]: Renamed them.

File: contrib/automation_tests/test_cases/top_down_tab.py (hunk 3 above)
Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/top_down_tab.py:125]: Done.

File: contrib/automation_tests/test_cases/top_down_tab.py (hunk 5 above)
Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/top_down_tab.py]: Done.

**Review by florian-kuebler — COMMENTED** [review]
Thanks for the comments.

**Review by dpallotti — APPROVED** [review]

**Pull request merged**